dashboards/coredns/coredns.json

```diff
@@ -0,0 +1,696 @@
+{
+  "name": "CoreDNS Metrics",
+  "description": null,
+  "pages": [
+    {
+      "name": "Overview",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 3,
+            "height": 6
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![CoreDNS Logo](https://i.imgur.com/2P3A2zF.png) \n- - -\n### Overview\n[coreDNS](https://github.com/coredns/coredns) is a fast and flexible DNS server/forwarder. The key word here is *flexible*, with CoreDNS you are able to do what you want with your DNS data by utilizing plugins. If some functionality is not provided out of the box you can add it by [writing a plugin](https://coredns.io/explugins).CoreDNS is a [Cloud Native Computing Foundation](https://cncf.io) graduated project.\n\n### About this Dashboard\n\nThis dashboard is based on standard `coredns` Prometheus metrics ingested using Prometheus [remote write](https://one.newrelic.com/marketplace/catalog-pack-details/detail?state=d8bbbb94-e24f-8ef8-a440-f815fc4b340a).  \n\nTo filter, type `prometheus_server` into the filter bar and select the Prometheus server name to scope the dashboard to that instance.\n\nCommon attributes used in the filter bar are:\n\n- `prometheus_server`\n- `instance`\n- `pod` (if it exists on the metric)\n\n\n"
```

Comment:
should there be a note here that the default filter on some widgets assumes instance port `9153`?